jbmc/regression/jbmc-strings/CProverStringToString/Main.java

```diff
@@ -0,0 +1,184 @@
+import org.cprover.CProverString;
+
+class Main {
+  public void testBooleanSuccess(boolean b) {
+    String s = CProverString.toString(b);
+    assert (b && s.equals("true")) || (!b && s.equals("false"));
+  }
+
+  public void testBooleanFailure() {
+    String s = CProverString.toString(true);
+    assert s.equals("false");
+  }
+
+  public void testCharSuccess(int choice) {
+    char c;
+    String r;
+
+    switch(choice) {
+    case 0:
+      c = 'a';
+      r = "a";
+      break;
+    case 1:
+      c = ' ';
+      r = " ";
+      break;
+    case 2:
+      c = '"';
+      r = "\"";
+      break;
+    case 3:
+      c = 'デ';
+      r = "デ";
+      break;
+    case 4:
+      c = Character.MIN_VALUE;
+      r = "\u0000";
+      break;
+    /*case 5:
+      c = Character.MAX_VALUE;
+      r = "\uFFFF";
+      break;*/
```

Comment:
why is this commented out? :coffee:

Reply:
This case does not work. I have to investigate why.